route/build/src/ncio_utils.f90

```diff
@@ -198,6 +199,25 @@ subroutine get_nc_dim_len(fname,           &  ! input: filename
 
  end subroutine
 
+ ! *********************************************************************
+ ! subroutine: check if variable exists
+ ! *********************************************************************
+ function check_variable(ncid, vname)
```

Comment:
I like having a subroutine for this kind of functionality.

Reply:
Would it be useful to add calls to this in other places as well, so that you could check if variables exist.